### DISCUSSION TITLE
Are these Lists included in the sanctions list?

### DISCUSSION THREAD

**Discussion by Thridev** [moov-io/watchman#564] [Q&A]
Hi All,
Does anyone know if these lists are included in the current sanction lists that Watchman runs through or are these already a part of the lists mentioned in the README ? Thank you
1. FinCEN 311
2. CIA’s Chief's of State and Cabinet Members of Foreign Governments (PEP)

**Comment by adamdecaf** [comment]
By PEP do you mean the Politically Exposed Persons list? That's in the US CSL.

I don't believe the FinCen lists are public.
